src/index.ts

```diff
@@ -38,6 +38,9 @@ export const Before = methods.Before
 export const BeforeAll = methods.BeforeAll
 export const BeforeStep = methods.BeforeStep
 export const defineParameterType = methods.defineParameterType
+/**
+ * @deprecated use `Given`, `When` or `Then` instead; see <https://github.com/cucumber/cucumber-js/issues/2043>
+ */
```

Comment:
Instead of a comment, should we use util.deprecate? https://nodejs.org/api/util.html#utildeprecatefn-msg-code

Reply:
Hmm probably both I think. The comment will give people feedback at the editor/linter level, and then this will flag at runtime. Seems like a good combination.

Reply:
(done)